Source/SlideController.swift

```diff
@@ -364,6 +371,11 @@ public class SlideController<T, N>: NSObject, UIScrollViewDelegate, ControllerSl
         guard pageIndex != currentIndex || forced else {
             return
         }
+        // Do not allow multiple shift calls,
+        // the problem is described in https://stackoverflow.com/a/16898199
+        guard !isForcedToSlide else {
```

Comment:
Please add `forced` argument check in this guard statement to let us force shift.